ajax/libs/proteic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "proteic",
+  "filename": "proteic.js",
+  "version": "0.0.27",
+  "description": "Streaming and static data visualization for the modern web",
+  "homepage": "https://proteic.js.org/",
+  "keywords": [
+    "d3",
+    "chart",
+    "data visualization",
+    "visualization",
+    "bigdata",
+    "data-visualization",
+    "dataviz"
+  ],
+  "author": "0xNacho",
```

Comment:
Wrong author? https://github.com/proteus-h2020/proteic/blob/0.0.30/package.json#L24-L26

Reply:
@extend1994 If you will go to commits the first commit was done by this contributor